### PR TITLE
[docs] Sort es/toctree.yml like en/toctree.yml

### PR DESCRIPTION
**Pull request by aaronjimv**
# What does this PR do?

I think that the file `es/_toctree.yml` is not aligned with `en/_toctree.yml`.

I would like to ask if it was this way intentionally, and if not the case, I would appreciate checking this change.

I kept this part the same because the `Performance and Scalability` section is not in the Spanish documentation:
```
- isExpanded: false
    sections:
    - local: debugging
      title: Debugging
    title: Rendimiento y escalabilidad
```
Thanks for your time.

Fixes # (issue)

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

@osanseviero @stevhliu

### CODE REVIEW

**Comment by aaronjimv** [comment]
I was doing a translation of `performance.md`, whose section is not in the Spanish documentation, and I got the impression that the file `es/_toctree.yml` is not aligned with `en/_toctree.yml`. I am open to any comments.

**Comment by stevhliu** [comment]
Hi the `es/_toctree` should be aligned with the English version, so please feel free to realign it! You can also create the `Performance and scalability` section for the Spanish docs if you'd like 😄

**Comment by aaronjimv** [comment]
Hi @stevhliu, thanks for replying, I add the `Performance and Scalability` section to `es/_toctree.yml` and I would like to check this new alignment.

File: docs/source/es/_toctree.yml [hunk 1]
```diff
@@ -20,58 +20,59 @@
...
-      - local: tasks/question_answering
-        title: Respuesta a preguntas
-      - local: tasks/language_modeling
-        title: Modelado de lenguaje
-      - local: tasks/summarization
-        title: Generación de resúmenes
-      - local: tasks/multiple_choice
-        title: Selección múltiple
-      title: Guías de tareas
+- section:
```
Comment (stevhliu) [inline docs/source/es/_toctree.yml:23]: This needs to be `sections` otherwise its not recognized :)

```suggestion
- sections:
```

Comment (aaronjimv) [reply docs/source/es/_toctree.yml]: Done, thanks!

**Comment by aaronjimv** [comment]
Continues in #28262

**Pull request closed without merging**